src/views/payment-sheet-views/card-view.js

```diff
@@ -665,6 +665,11 @@ function hideCardIcon(icon) {
 function isCardViewElement() {
   var activeId = document.activeElement && document.activeElement.id;
   var isHostedFieldsElement = document.activeElement instanceof HTMLIFrameElement && activeId.indexOf('braintree-hosted-field') !== -1;
+  var isPaypalCollectionNotice = document.activeElement.parentElement.className === 'braintree-form__notice-of-collection';
```

Comment:
we should add some safety for accessing these object properties

Reply:
done